core/eolearn/core/utils/testing.py

```diff
@@ -50,7 +52,7 @@ def generate_eopatch(
 ) -> EOPatch:
     """A class for generating EOPatches with dummy data."""
     config = config if config is not None else PatchGeneratorConfig()
-    supported_feature_types = [ftype for ftype in FeatureType if ftype.is_array()]
+    supported_feature_types = [ftype for ftype in FeatureType if ftype.is_array()].append(FeatureType.META_INFO)
```

Comment:
this is a clear mistake. Investigate why our tests did not catch it

Reply:
(`mypy` did, good boy :dog2: )

Reply:
> Investigate why our tests did not catch it 

supported_feature_types went to `None` so all is "supported" we do not have test for that

Reply:
if all is supported, why didnt our tests for "vector features are not allowed" detect that?

Reply:
ValueError come from wrong dimension assigned to feature

Reply:
Change error to `NotImplementedError`